test/components/generators/chat/test_hugging_face_local.py

```diff
@@ -42,6 +42,7 @@ def mock_pipeline_tokenizer():
     # Mocking the tokenizer
     mock_tokenizer = Mock(spec=PreTrainedTokenizer)
     mock_tokenizer.encode.return_value = ["Berlin", "is", "cool"]
+    mock_tokenizer.pad_token_id = 100
```

Comment:
`PreTrainedTokenizer` in new versions of `transformers` does not have a default `pad_token_id`, so I'm setting it in the Mock.
(In any case, we are confident that our component works properly also thanks to an integration test.)